web/modules/custom/bglobal_angular/components/info_sessions_list/component.ts

```diff
@@ -0,0 +1,55 @@
+import {Component} from "@angular/core";
+import {ElementRef} from '@angular/core';
+import {Inject} from '@angular/core';
+import {Http} from '@angular/http';
+import 'rxjs/add/operator/map';
+
+@Component({
+  moduleId: __moduleName,
+  selector: "info-sessions-list",
+  templateUrl: 'template.html'
+})
+
+export class InfoSessionsList {
+  public cert: string = 'all';
+  public area: string = 'all';
+  public baseUrl: string = 'https://voices.berkeley.edu/info-sessions/json/';
+  
```

Comment:
Technically, any attribute using `this` should be set here -- e.g. `view`, `sessions`, `limit`